src/invidious/views/channel/about.ecr

```diff
@@ -0,0 +1,45 @@
+<% content_for "header" do %>
+<title><%= channel.author %> - Invidious</title>
+<link rel="stylesheet" href="/css/channel.css?v=<%= ASSET_COMMIT %>">
+<% end %>
+
+<% content_type =  5 %>
+<%= rendered "components/channels/channel-information" %>
+
+<div class="pure-g">
+    <% stats_style_append = ""%>
+    <% if !channel.country.empty? %>
+        <div class="pure-u-1 pure-u-md-3-4">
+            <div class="h-box">
+                <h3> <%= translate(locale, "channel_about_page_details_section_header") %> </h3>
+                <p>Country: <%=channel.country%></p> <!-- Translate Country Names -->
+            </div>
+        </div>
+    <% else %>
+        <% stats_style_append = "style=\"order: 1;\"" %>
```

Comment:
What's that for? Can't it be done with CSS instead (with `:first-child` pseudo-class)?